CIAC-12314: Add Array Support to Rasterize (#39627)
demisto/content: Packs/rasterize/ReleaseNotes/2_1_4.md

#### Integrations

##### Rasterize

> - Updated the Rasterize integration to support arrays in the war room.

Reviewer: ```suggestion
Updated the Rasterize integration to support arrays in the War Room.
```

##### Rasterize

> Updated the Rasterize integration to support arrays in the War Room.

Reviewer 2: ```suggestion
Updated the Rasterize integration to support a list of URLs as an argument.
```
